lib/iso3166Data.js

```diff
@@ -1880,5 +1880,389 @@ module.exports = [
 		country_name: 'Zimbabwe',
 		mobile_begin_with: ['71', '73', '77'],
 		phone_number_lengths: [9]
-	}
+	},
+	{
+        "alpha2": "SX",
+        "alpha3": "SXM",
+        "country_code": "1",
+        "country_name": "Sint Maarten",
+        "mobile_begin_with": [
+            "721"
+        ],
+        "phone_number_lengths": [
+            10
+        ]
+	},
+	{
+        "alpha2": "PE",
+        "alpha3": "PER",
+        "country_code": "51",
+        "country_name": "Peru",
+        "mobile_begin_with": [
+            "9",
+            "6"
+        ],
+        "phone_number_lengths": [
+            9,
+            8
+        ]
+	},
+	{
+        "alpha2": "MY",
+        "alpha3": "MYS",
+        "country_code": "60",
+        "country_name": "Malaysia",
+        "mobile_begin_with": [
+            "1",
+            "6"
+        ],
+        "phone_number_lengths": [
+            9,
+            10,
+            8
+        ]
+	},
+	{
+        "alpha2": "US",
+        "alpha3": "USA",
+        "country_code": "1",
+        "country_name": "United States",
+        "mobile_begin_with": [
+            "201",
```

Comment:
please reduce no. of lines similar to other countries

Reply:
ok,thanks